onnxruntime/core/providers/openvino/openvino_provider_factory.cc

```diff
@@ -39,6 +39,7 @@ struct OpenVINOProviderFactory : IExecutionProviderFactory {
   std::string precision_;
   bool enable_npu_fast_compile_;
   size_t num_of_threads_;
+  std::string load_config_;
```

Comment:
here i think device_type and cache_dir is being forced set to "" if nullptr. https://github.com/intel/onnxruntime/blob/cfa45df6b5060af6327a98a625eb9fe74580f56c/onnxruntime/core/providers/openvino/openvino_provider_factory.cc#L31

Reply:
fixed